src/System.Private.CoreLib/src/System/Threading/ClrThreadPoolEventSource.cs

```diff
@@ -15,38 +15,48 @@ public sealed class ClrThreadPoolEventSource : EventSource
         private const string WorkerThreadAdjustmentStatsEventMessage = "Duration=%1;%nThroughput=%2;%nThreadWave=%3;%nThroughputWave=%4;%nThroughputErrorEstimate=%5;%nAverageThroughputErrorEstimate=%6;%nThroughputRatio=%7;%nConfidence=%8;%nNewControlSetting=%9;%nNewThreadWaveMagnitude=%10";
 
         // We have these constants here because EventTask, EventOpcode, and EventKeywords are public APIs, and we don't want to expose these values to the end user to use in their events.
```

Comment:
This comment does not make sense anymore.

The comments on these in other event sources are "// The task definitions for the ETW manifest." or "// This name is important for EventSource". Do the same here?

The comments